src/_pages/ru/index.ejs

```diff
@@ -49,6 +49,40 @@ title: Главная
             </div>
         </div>
     </div>
+    <h2 class="c-section-head">как работает приложение</h2>
+    <div class="c-app">
+      <div class="c-app__desktop-img">
+        <img id="screenshot-app"src="/images/screenshot-app-1.png" alt="screenshot-app">
+      </div>
+      <div class="swiper c-app__imgs u-hidden">
+        <div class="swiper-wrapper">
+          <div class="swiper-slide c-app__img">
+            <img id="screenshot-app-1" src="/images/screenshot-app-1.png" alt="screenshot-app-1">
+          </div>
+          <div class="swiper-slide c-app__img">
+            <img id="screenshot-app-2" src="/images/screenshot-app-2.png" alt="screenshot-app-2">
```

Comment:
In `alt` we should put some description of [what is actually shown on image](https://weblind.ru/inner.html#image-content).